common/messages/messages_en.go

```diff
@@ -219,6 +219,13 @@ var messagesEn = map[string]util.Message{
 		RC:          400,
 		Action:      "Please verify that the start volume ID is correct and whether you have access to the volume ID.",
 	},
+	"InvalidServiceSession": {
+		Code:        "InvalidServiceSession",
+		Description: "The Service Session was not found due to error while generating IAM token for Sesssion Service.",
```

Comment:
can it be `The Service Session was not found due to error while generating IAM token`